### PR TITLE
errorcode: Add TiFlashServerBusy and TiFlashServerTimeout.

### PR DESCRIPTION
**Pull request by LittleFall**
### What is changed, added or deleted? (Required)

Add errorcodes TiFlashServerBusy and TiFlashServerTimeout.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: need translate to English
- Other reference link(s): https://github.com/pingcap/tidb/pull/21926

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lzmhhh123 — COMMENTED** [review]
LGTM

**Comment by LittleFall** [comment]
@TomShawn PTAL

File: error-codes.md [hunk 1]
```diff
@@ -347,6 +347,14 @@ TiDB 兼容 MySQL 的错误码，在大多数情况下，返回和 MySQL 一样
...
 
     TiKV 无法处理这条 raft log，请检查 TiKV Server 状态/监控/日志。
+    
+* Error Number: 9012
+
+    请求 TiFlash 超时，请检查 TiFlash Server 状态/监控/日志以及 TiDB Server 与 TiFlash Server 之间的网络。
+
+* Error Number: 9013
+
+    TiFlash 操作繁忙，一般出现在数据库负载比较高时，请检查 TiFlash Server 状态/监控/日志。
```
Comment (TomShawn) [inline error-codes.md:357]:
```suggestion
    TiFlash 操作繁忙。该错误一般出现在数据库负载比较高时。请检查 TiFlash Server 的状态/监控/日志。
```

File: error-codes.md [hunk 2]
```diff
@@ -347,6 +347,14 @@ TiDB 兼容 MySQL 的错误码，在大多数情况下，返回和 MySQL 一样
 * Error Number: 9010
 
     TiKV 无法处理这条 raft log，请检查 TiKV Server 状态/监控/日志。
+    
+* Error Number: 9012
+
+    请求 TiFlash 超时，请检查 TiFlash Server 状态/监控/日志以及 TiDB Server 与 TiFlash Server 之间的网络。
```
Comment (TomShawn) [inline error-codes.md:353]:
```suggestion
    请求 TiFlash 超时。请检查 TiFlash Server 状态/监控/日志以及 TiDB Server 与 TiFlash Server 之间的网络。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**